src/server/utils/watchLocalModules.js

```diff
@@ -43,12 +44,13 @@ export default function watchLocalModules() {
 
       const moduleMap = JSON.parse(fs.readFileSync(moduleMapPath, 'utf8'));
 
-      const moduleData = moduleMap.modules[moduleNameChangeDetectedIn];
+      const moduleData = addBaseUrlToModuleMap(moduleMap).modules[moduleNameChangeDetectedIn];
       const oneAppDevCdnAddress = `http://${getIp()}:${process.env.HTTP_ONE_APP_DEV_CDN_PORT || 3001}`;
 
       moduleData.browser.url = moduleData.browser.url.replace('[one-app-dev-cdn-url]', oneAppDevCdnAddress);
       moduleData.legacyBrowser.url = moduleData.legacyBrowser.url.replace('[one-app-dev-cdn-url]', oneAppDevCdnAddress);
       moduleData.node.url = moduleData.node.url.replace('[one-app-dev-cdn-url]', oneAppDevCdnAddress);
+      moduleData.baseUrl = moduleData.baseUrl.replace('[one-app-dev-cdn-url]', oneAppDevCdnAddress);
```

Comment:
Is this line tested at all?

Reply:
Should be covered by the existing test cases that Michael modified

Reply:
were they failing before?